data-prep/README.md

```diff
@@ -15,9 +15,59 @@ Data Cleaning and Munging
 
 A number of cleaning steps are necessary to use the raw data.
 
-### Project Filtering
+### Consumption Data Cleaning
+
+**Elec Traces**
+
+* Remove duplicates
+  * No duplicates in monthly data
+* Add leading zeroes to `PREM_ID` where missing (length == 8 or 9) in monthly traces.
+
+* 7702 elec monthly traces with unique SA_ID
+
+**Gas Traces**
+
+* Remove duplicates
```

Comment:
For gas:

'gas-daily': ['Service Point', 'Measurement Date']
'gas-monthly': ['SA_ID', 'CDT__1'],